cmd/auth-rest/startcmd/test/data/providers.yaml

```diff
@@ -10,6 +10,14 @@ providers:
     clientID: auth
     clientSecret: auth-secret
     name: Demo OIDC
-    signUpLogoURL: https://localhost:8070/static/images/signup_logo.svg
-    signInLogoURL: https://localhost:8070/static/images/signin_logo.svg
-    order: 1
\ No newline at end of file
+    signUpIconURL:
+      {
+        en: https://localhost:8070/static/images/sign-up-icon-en.svg,
+        fr: https://localhost:8070/static/images/sign-up-icon-fr.svg,
+      }
```

Comment:
Can we use same structure as in bdd-test for yaml file https://github.com/trustbloc/auth/pull/207/files#diff-83705247bf240739c351e7a6ccce654cbd6bb1dbe5f4a147820a78049547686fR14-R17 ?